### PR TITLE
Better dependency management (poetry)#12

### PR DESCRIPTION
**Pull request by bq777xCube**

### CODE REVIEW

**Comment by atsanda** [comment]
I got an error while running `poetry install`
movie-surfer/movie_surfer does not contain any element

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "movie-surfer"
+version = "0.1.0"
+description = ""
+authors = ["Islam Iangurazov <islamyang123@gmail.com>"]
+readme = "README.md"
+packages = [{include = "movie_surfer"}]
```
Comment (atsanda) [inline pyproject.toml]:
```suggestion
packages = [{include = "moviesurfer"}]

[tool.poetry.scripts]
moviesurfer = 'moviesurfer.cli.main:main'
```

Comment (atsanda) [reply pyproject.toml]: to make CLI work

**Review by atsanda — CHANGES_REQUESTED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+[tool.poetry.dependencies]
+python = "^3.10"
+click = "^8.1.3"
+sphinx = "^7.0.0"
```
Comment (atsanda) [inline pyproject.toml]: we only need it to build documentation
it probably make sense to make another group of dependencies and place it there
what do you think?

File: pyproject.toml (hunk 1 above)
Comment (bq777xCube) [reply pyproject.toml]: agreed.

File: pyproject.toml (hunk 2 above)
Comment (bq777xCube) [reply pyproject.toml]: agreed.

**Review by bq777xCube — COMMENTED** [review]
Fixed. Please check.

**Review by atsanda — APPROVED** [review]

**Pull request merged**